tasks/libs/releasing/version.py

```diff
@@ -61,11 +61,13 @@ def _create_version_from_match(match):
     return version
 
 
-def check_version(agent_version):
+def check_version(agent_version, agent6=False):
     """Check Agent version to see if it is valid."""
-    version_re = re.compile(r'7[.](\d+)[.](\d+)(-rc\.(\d+))?')
+
+    lead = '6' if agent6 else '7'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ maybe `major` instead of `lead`?